test/puppetlabs/puppetdb/testutils.clj

```diff
@@ -26,6 +27,17 @@
             [clojure.set :refer [difference]]
             [puppetlabs.puppetdb.test-protocols :as test-protos]))
 
+(defn ordered-matches?
```

Comment:
Yeah, we are, but I think I still have a version without it (using the big regex) if we want to switch, or we could decide we just don't care about checking the err file as much, and do something even simpler if we like.